sites/kit.svelte.dev/src/routes/index.svelte

```diff
@@ -80,6 +80,7 @@
 
 	<div slot="how">
 		<pre><code>
+// You'll need Node.js v14 or higher
```

Comment:
This isn't JS so should use bash-style commenting
```suggestion
# You'll need Node.js v14 or higher
```

Reply:
Aah right, fixed now.